manifests/blog.pp

```diff
@@ -139,7 +125,7 @@
 
   if $use_supervisor {
     supervisor::program { "ghost_${blog}":
-      command        => "node ${home}/index.js",
+      command        => "nodejs ${home}/index.js",
```

Comment:
So whats weird with this is on my Docker image I get:

```
/usr/bin/env: node: No such file or directory
npm WARN This failure might be due to the use of legacy binary "node"
npm WARN For further explanations, please read
/usr/share/doc/nodejs/README.Debian
```

But it works fine on my Vagrant image...